content/md/administration/catalog-volume-monitoring_en.md

```diff
@@ -22,16 +22,16 @@ This screen is only available starting the 2.3 version of the PIM.
 
 To access this amazing (;)) screen, follow these very simple steps:
 1. Click on the `Activity` menu
-1. Click on the `Catalog volume monitoring` menu entry
-1. Enjoy!
+2. Click on the `Catalog volume monitoring` menu entry
```

Comment:
You don't need to change that. Markdown automatically adapts the numbers correctly. :)